SPECS/debugedit/debugedit.spec

```diff
@@ -1,36 +1,45 @@
 Summary:      Debugedit - obtain debug information from binaries.
 Name:         debugedit
 Version:      5.0
-Release:      2%{?dist}
+Release:      3%{?dist}
 License:      GPLv3+
 URL:          https://sourceware.org/debugedit/
 Vendor:       Microsoft Corporation
 Distribution: Mariner
 Source0:      https://sourceware.org/ftp/%{name}/%{version}/%{name}-%{version}.tar.xz
+Patch0:       BUG-28161.patch
+BuildRequires: automake
+BuildRequires: binutils
+BuildRequires: help2man
 
 %description
 %{summary}
 
 %prep
-%autosetup
+%autosetup -p1
 
 %build
+autoreconf -fvvi
 %configure
 %make_build
 
 %install
 %make_install
 
 %check
-%make_build_check
+make check
+cat tests/testsuite.log
```

Comment:
I think that the `cat` command will mask any errors from `make check`. Can you do something like the [gfs2-utils.spec does here](https://github.com/microsoft/CBL-Mariner/blob/04daa7acedc18ef55143e3a8382308349565b17d/SPECS/gfs2-utils/gfs2-utils.spec#L49C43-L49C43)?

Reply:
Done :)